.github/workflows/build.yml

```
@@ -30,6 +30,29 @@ jobs:
         make VERBOSE=1
         make test VERBOSE=1
 
+  # The other test jobs build with BUILD_TOOLS=OFF, and the packaging jobs build
+  # fpcalc against a pinned FFmpeg. Nothing builds the FFmpeg audio reader
+  # against the system FFmpeg, which is the configuration distributions use.
+  test-linux-tools:
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)